### PR TITLE
login endpoint + store

### PR DESCRIPTION
**Pull request by ertip4geek**

### CODE REVIEW

File: src/api/routes.py [hunk 1]
```diff
@@ -63,3 +63,41 @@ def create_user():
...
+    # create_access_token() function is used to actually generate the JWT.
+@api.route("/login", methods=["POST", "GET"])
+def login():
+    body = request.get_json()
+    if body is None:
+        return jsonify({"error": "Body is empty or null"}), 400
+    email = body['email']
+    password = body['password']
+    # 1. read mail & pass
+    # email = request.json.get("email", None)
```
Comment (emewarp) [inline src/api/routes.py:76]: you should remove the commented codes

Comment (ertip4geek) [reply src/api/routes.py:76]: done

File: src/api/routes.py [hunk 2]
```diff
@@ -63,3 +63,41 @@ def create_user():
...
+    user = User.lookup(email)
+    # 2 check error
+    if user and check_password_hash(user.password, password):
+    # 3. return token
+        access_token = create_access_token(identity=email)
+        return jsonify({'token' : access_token}), 200
+    else:
+        return {'error': 'user and pass not valid'}, 400
+
+@api.route('/refresh', methods=['POST'])
```
[the remark this answers is not in the document]
Comment (ertip4geek) [reply src/api/routes.py:88]: fixed

File: src/front/js/store/flux.js [hunk 3]
```diff
@@ -1,49 +1,20 @@
...
+    },
+    actions: {
+      // Use getActions to call a function within a fuction
+      getNannies: () => {
+        fetch(
+          "https://nannyhub-nannyshub-79jxz11ahr8.ws-eu38.gitpod.io/api/nannies"
+        )
+          .then((resp) => resp.json())
+          .then((data) => setStore({ nanny: data }))
+          .catch((error) => console.log("Baby is crying", error));
```
[the remark this answers is not in the document]
Comment (ertip4geek) [reply src/front/js/store/flux.js:14]: changed

**Pull request merged**